packages/thumbprint-react/components/Alert/index.module.scss

```diff
@@ -0,0 +1,66 @@
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+
+.tpAlert {
+    display: flex;
+    font-size: $tp-font__body__2__size;
+    line-height: $tp-font__body__2__line-height;
+    text-align: left;
+    padding: 12px $tp-space__3;
+    color: $tp-color__white;
+    background: $tp-color__gray;
+    font-weight: 700;
+    border-radius: $tp-border-radius__base;
+
+    & a {
```

Comment:
Do we need to target an element here? I would hope our reset sheet for links would handle this. How does this interact with atomic classes, which use `!important`?

Reply:
It appears we do need to target the anchor here. Without it the style gets overridden by the default.